src/cometh/rental-protocol/rental-started-handler.ts

```diff
@@ -0,0 +1,77 @@
+import { log } from '@graphprotocol/graph-ts'
+import { Nft, Rental, RentalOffer } from '../../../generated/schema'
+import { generateNftId } from '../../utils/misc'
+import { COMETHSPACESHIP } from '../../utils/constants'
+import { RentalStarted } from '../../../generated/ComethRentalProtocol/ComethRentalProtocol'
+import { SPACESHIP_ADDRESS } from '../../utils/addresses'
+
+/**
+ *     event RentalStarted(
+ *        uint256 indexed nonce,
+ *        address indexed lender,
+ *        address indexed tenant,
+ *        address token,
+ *        uint256 tokenId,
+ *        uint64 duration,
+ *        uint16 basisPoints,
+ *        uint256 start,
+ *        uint256 end
+ *    );
+ */
+
+export function handleRentalStarted(event: RentalStarted): void {
```

Comment:
After creating the rental, shouldn't we also add it to the `rentalHistory`? Or only add it to the history once it ends?

Reply:
No because rentalHistory is using @derivedFrom. So the graph will index it by the ID automatically. 

But that reminds me that we can use the same strategy with RentalOfferHistory and save us processing time.